components/_common_/Form/FormInput/FormInput.js

```diff
@@ -40,7 +40,8 @@ class FormInput extends Component {
       },
       () => {
         if (props.onChange) {
-          props.onChange(state.text, state.isValid);
+          const { text, isValid: isValidAfterSetState } = this.state;
```

Comment:
you can simply destructure `state` here - it'll resolve the lint error

Reply:
you could, but it would increase the number of parameters the `onChange` function is called. With this implementation, I'm totally aware of what will be passed.

Reply:
Sorry. Maybe I miscommunicated.

Above, we've destructued state via `const { props, state } = this;`

by also doing `const { text, isValid: isValidAfterSetState } = this.state;` were not using the state reference we made earlier. I'm simply saying do this instead so that you dont have a lint error:

```
const { text, isValid: isValidAfterSetState } = state;
```